src/tests/eosjs-jsonrpc.test.ts

```diff
@@ -606,6 +606,49 @@ describe('JSON RPC', () => {
         expect(fetch).toBeCalledWith(endpoint + expPath, expParams);
     });
 
+    it('calls send_readonly_transaction', async () => {
```

Comment:
this test file is super bloated because we keep making the same huge constants in multiple tests like signatures below. Instead they should just be declared outside the scope of individual tests so they can be used everywhere rather than redeclaring them all over the place. This would require a light refactor and I won't block the PR based on that but it would be good for cleanup.

Reply:
Making me work for this PR :) . I gave it a go , and consolidated the signatures, serializedtrasnactions, accountname, and blockid. Saved about 30 lines.